features/notifications/notifications-init.ts

```
@@ -21,70 +24,94 @@ export function configureForegroundNotificationBehavior() {
+    captureError(
+      new NotificationError({
+        error,
+      }),
+    )
+  } finally {
+    // Let's always show the notification anyway
     return {
       shouldShowAlert: true,
       shouldPlaySound: true,
       shouldSetBadge: true,
     }
```

Biome (1.9.4)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)